src/client/core/utils/shared/values/axis-values.ts

```diff
@@ -44,4 +44,11 @@ export default class AxisValues<T> implements AxisValuesData<T> {
 
         return this;
     }
+
+    public round (this: AxisValues<number>, fn = Math.round): AxisValues<number> {
```

Comment:
`this` is a bad name for the function argument. Rename here and in others places in this file.

`public round (axisValues: AxisValues<number>, roundFn = Math.round): AxisValues<number>`

Reply:
`this` is not a function parameter. This is the type of `this` context. We cannot apply this function to `AxisValues<boolean>`. Therefore, we define the type of `this`.

Reply:
Ok